workbench-core/accounts/src/models/costCenters/costCenter.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+export interface CostCenter {
```

Comment:
you can just do an infer and save some code 
`export type CostCenter = z.infer<typeof CostCenterParser >;`

Reply:
Good call, updating